src/skin/css/firstRun.css

```diff
@@ -788,6 +788,13 @@ body {
   background-image:linear-gradient(#DFDFE6, #DFDFE6 30%, #fff 30%);
   padding:4rem 0;
 }
+#pb-privacy-policy, #pb-privacy-policy a {
+  color:#707070;
+  font-family:"OpenSans-Light",sans-serif;
+  font-size:0.7rem;
+  text-align:center;
```

Comment:
May not be needed once HTML typo is fixed.

Reply:
I ended up removing the class as it wasn't doing anything and this was still needed.